### DISCUSSION TITLE
Import from New World Scheduler, soon?

### DISCUSSION THREAD

**Discussion by luisenriquereyes** [sws2apps/organized-app#2936] [Q&A] [answered]
Greetings 👋  I am very interested in moving away from New World Scheduler to your Organized App. I see in the FAQ that importing from other popular apps is on the road map.  Any estimate on when importing from New World Scheduler will be available?

**Comment by fschein** [comment]
I would love to have this function to import from Hourglass, but my anxiety won't allow me to wait. I'll enter everything again; we haven't fully started using HG yet. Good thing the elders here took their time approving its use :laughing:

**Comment by ux-git** [comment, marked as answer]
Hello 👋 

Thank you for your interest in the Organized app!

Yes, we’re actively exploring ways to implement a feature that allows importing data from other apps, as well as enabling data exports for use with other applications of user's choice. While we have many features currently in development and even more planned, software timelines can be challenging to predict. Therefore, while this functionality is a high priority, it’s likely to be months rather than weeks away. We want to ensure that everything works seamlessly and reliably, and we’re committed to delivering it as soon as possible.

Thank you for your enthusiasm and patience! 😊

**Reply by luisenriquereyes** [reply]
I you all need a beta tester to help test development importing data from NW Scheduler into Organized, my hand is up 🤚😁

**Reply by ux-git** [reply]
@luisenriquereyes could you drop a message in [our Telegram chat](https://t.me/+2VCKmqGLYzIxZTRi)? We'd like to get in touch with you about some assistance with this feature 🙂
